src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1793,21 +1793,18 @@ public function CopyMLTRecords($oField, $sourceRecordID)
     {
         $mltTableName = $oField->GetMLTTableName();
         $oField->oTableRow->id = $sourceRecordID;
-        $oFieldType = $oField->oDefinition->GetFieldType();
-        if ('CMSFIELD_DOCUMENTS' == $oFieldType->sqlData['constname']) {
```

Comment:
This was removed by intention?

Reply:
yes - there is no need to load the foreign table, so we no longer need to know what it is called. We only need the ID, which we can get from the target_id field